src/web/reducers/flows.js

```diff
@@ -160,6 +103,60 @@ function updateCurrentHash(state) {
   return { ...state, currentHashes: computeFlowsHash(state) }
 }
 
+function createSnapshot(state) {
```

Comment:
Was this only moved or changed?

Reply:
Can we please always use fat arrow functions unless we really need them to be normal `function`s? Arrow functions are easier to reason about :)

Reply:
Agreed but sometimes I find hoisting to help make things cleaner (e.g. you don't have to surface internal methods with the public interface), would like to keep a window open for such cases :)

Reply:
This one was only moved to the bottom due to new eslint-rules, so I think we could refactor it out of scope of current task.